lib/ethpy/ethpy/hyperdrive/state/conversions.py

```diff
@@ -31,15 +28,15 @@ def snake_to_camel(snake_string: str) -> str:
 
 
 def dataclass_to_dict(
```

Comment:
This function is basically "hypertypes_or_fixedpoint_state_to_contract"; in other words, it converts the hypertypes _or_ fixedpoint versions of pool info, pool config, & checkpoint into a dictionary that should look exactly like what the contract web3 read call returns. I think it makes sense to keep this iwth the other conversions. Then you can delete this file all-together and move the test as well.

Reply:
ok sounds good.